examples/python/mapping_example.py

```diff
@@ -474,6 +472,24 @@ def distance2_CRy_decomp(
 
 # Also note that higher performance solutions are coded straight into the TKET c++ codebase. This provides advantages, including that Circuit construction and substitution is unncessary (as with python) as the circuit can be directly modified, however the ability to produce prototypes at the python level is very helpful. If you have a great python implementation but are finding some runtime bottlenecks, why not try implementing it straight into TKET (the code is open source at https://github.com/CQCL/tket).
 
+# The LexiLabellingMethod is only labelling qubits that are used by two qubit gates, to labelled qubits with only single qubits gates or qubits without any gates at all this should be combined with a pass for a general placement first.
```

Comment:
The LexiLabellingMethod only labels qubits that are involved in multi-qubit interactions. If a qubit has only single qubit gates or no gates at all this method should be combined with a pass for a general placement first.